internal/versions/versions.go

```diff
@@ -165,7 +165,7 @@ func (h *handler) GetReleaseVersion(openshiftVersion string) (string, error) {
 	}
 
 	if h.openshiftVersions[versionKey].ReleaseVersion == nil {
-		return h.GetOCPSemVer(openshiftVersion)
+		return "", errors.Errorf("Release version was missing for openshift version %s", versionKey)
```

Comment:
Is that a user error or internal error?

Reply:
It's an unexpected internal error since 'ReleaseVersion' is a required property. 
So for example the same as GetRHCOSVersion:
https://github.com/openshift/assisted-service/blob/9e1f1680a249925e9a020485d410c4507a816c09/internal/versions/versions.go#L142